qcodes/tests/parameter/test_parameter_scale_offset.py

```diff
@@ -4,7 +4,7 @@
 from hypothesis import given, event, settings
 import numpy as np
 
-from qcodes.instrument.parameter import Parameter
+from qcodes.instrument.parameter import Parameter, DelegateParameter
```

Comment:
Could you remove this import again since its not used

Reply:
Good point! Missed that, I'll remove it.

Question: what do you do with the slight decrease in coverage?

Reply:
Looks like the coverage is now back to baseline. The codecov tool can sometimes be a bit flaky. In general I would check on codecov where the drop is coming from and then decide if that is important and how to best write a test for that